.github/workflows/runas.yml

```diff
@@ -49,6 +49,7 @@ jobs:
             build-essential \
             llvm-14-dev libclang-14-dev clang-14 \
             gcc \
+            libkrb5-dev \
```

Comment:
I'm concerned about this new requirement.

If this is now required for pgrx extensions it will most likely break everyone's CI.  This isn't something I want to do.

Reply:
Can we do this such that kerberos isn't a required system dependency?

Reply:
The Postgres installed in the action is compiled `--with-openssl` so `libssl-dev` is installed on the line below this one.